examples/v1alpha2/basic-http.yaml

```diff
@@ -1,5 +1,5 @@
-kind: GatewayClass
-apiVersion: gateway.networking.k8s.io/v1alpha2
+apiVersion: GatewayClass
+kind: gateway.networking.k8s.io/v1alpha2
```

Comment:
something went wrong here. Applicable to multiple resources.

Reply:
I knew something looked off :slightly_smiling_face: should be fixed now